examples/kubernetes/kuma-demo/README.md

```diff
@@ -0,0 +1,235 @@
+Resolving akamai.bintray.com (akamai.bintray.com)... 104.93.1.149
+Connecting to akamai.bintray.com (akamai.bintray.com)|104.93.1.149|:443... connected.
+HTTP request sent, awaiting response... 200 OK
+Length: 42892462 (41M) [application/gzip]
+Saving to: ‘kuma-0.2.2-darwin-amd64.tar.gz’
+
+kuma-0.2.2-darwin-amd64.tar.g 100%[===============================================>]  40.91M  2.61MB/s    in 20s
+
+2019-10-13 05:54:08 (2.09 MB/s) - ‘kuma-0.2.2-darwin-amd64.tar.gz’ saved [42892462/42892462]
+```
+
+3. Unbundle the files to get the following components:
+
+```
+$ tar xvzf kuma-0.2.2-darwin-amd64.tar.gz
+x ./
+x ./conf/
+x ./conf/kuma-cp.conf
+x ./bin/
+x ./bin/kuma-dp
+x ./bin/envoy
+x ./bin/kuma-tcp-echo
+x ./bin/kumactl
+x ./bin/kuma-cp
+x ./README
+x ./LICENSE
+```
+
+4. Go into the /bin directory where the kuma components will be:
+
+```
+$ cd bin && ls
+envoy   kuma-cp   kuma-dp   kuma-tcp-echo kumactl
+```
+
+5. Install the control plane using `kumactl`
+
+```
+$ kumactl install control-plane | kubectl apply -f -
+namespace/kuma-system created
+secret/kuma-injector-tls-cert created
+secret/kuma-sds-tls-cert created
+secret/kuma-admission-server-tls-cert created
+configmap/kuma-injector-config created
+serviceaccount/kuma-control-plane created
+customresourcedefinition.apiextensions.k8s.io/dataplaneinsights.kuma.io created
+customresourcedefinition.apiextensions.k8s.io/dataplanes.kuma.io created
+customresourcedefinition.apiextensions.k8s.io/meshes.kuma.io created
+customresourcedefinition.apiextensions.k8s.io/proxytemplates.kuma.io created
+customresourcedefinition.apiextensions.k8s.io/trafficlogs.kuma.io created
+customresourcedefinition.apiextensions.k8s.io/trafficpermissions.kuma.io created
+clusterrole.rbac.authorization.k8s.io/kuma:control-plane created
+clusterrolebinding.rbac.authorization.k8s.io/kuma:control-plane created
+role.rbac.authorization.k8s.io/kuma:control-plane created
+rolebinding.rbac.authorization.k8s.io/kuma:control-plane created
+service/kuma-injector created
+service/kuma-control-plane created
+deployment.apps/kuma-control-plane created
+deployment.apps/kuma-injector created
+mutatingwebhookconfiguration.admissionregistration.k8s.io/kuma-admission-mutating-webhook-configuration created
+mutatingwebhookconfiguration.admissionregistration.k8s.io/kuma-injector-webhook-configuration created
+```
+
+Check the pods are up and running by checking the `kuma-system` namespace
+
+```
+$ kubectl get pods -n kuma-system
+NAME                                  READY   STATUS    RESTARTS   AGE
+kuma-control-plane-7bcc56c869-lzw9t   1/1     Running   0          70s
+kuma-injector-9c96cddc8-745r7         1/1     Running   0          70s
+```
+
+In the following steps, we will be using the pod name of the `kuma-control-plane-*************` pod. Please replace any `{KUMA_CP_POD_NAME}` with your pod name.
```

Comment:
Don't leave setting `KUMA_CP_POD_NAME` up to the users.

At least, provide a copy-and-paste example how to set it, e.g.

```
KUMA_CP_POD_NAME=$( kubectl -n kuma-system get pods -l app=kuma-control-plane -o=jsonpath='{.items[0].metadata.name}' )
```